src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,153 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.lang.String;
+import java.util.Scanner;
+
+public class Duke {
+    public static final int MAX_INT = 100;
+    private static Task[] tasks = new Task[MAX_INT];
+
+    public static void main(String[] args) {
+        // Level 4
+        displayDuke();
+        helloMessage();
+        commandsAvailable();
+
+        Scanner input = new Scanner(System.in);
+        int listIndex = 0;
+
```

Comment:
May consider write a method to print messages in each case so the main function body can be shortened and the logic can be easier to understand.